DESCRIPTION

```diff
@@ -30,37 +30,37 @@ Depends:
 Imports:
     assertthat,
     curl,
+    dplyr (>= 1.1.0),
     fields,
     frictionless,
     ggplot2,
     glue,
     graphics,
+    jsonlite,
     lubridate,
     lutz,
-    suntools,
     methods,
     raster,
     readr,
     rhdf5,
-    dplyr (>= 1.1.0),
+    rlang,
     sp,
     stats,
+    suntools,
     tidyr,
     utils,
-    viridisLite,
     viridis,
-    rlang,
-    jsonlite
+    viridisLite
 Suggests:
-    vol2birdR,
     aws.s3,
     ggmap (>= 3.0.0),
     knitr,
-    tidyselect,
     rmarkdown,
+    sf,
     testthat (>= 3.0.0),
+    tidyselect,
     vdiffr,
-    sf
+    vol2birdR
```

Comment:
Several collaborators wanted to keep this as a Suggest, and checks were added to all functions that use vol2birdR that prompt the user to install it when needed. Can't we keep it like that?

Reply:
Yes it seems so